### DISCUSSION TITLE
修改译文字体大小

### DISCUSSION THREAD

**Discussion by geekhuashan** [bookfere/Ebook-Translator-Calibre-Plugin#313] [Q&A] [answered]
翻译文字是否可选 大小为80% 

不知道怎么设置？

实在 CSS selector for priority elements 那里设置吗

应该怎么写啊

**Comment by bookfere** [comment]
建议用 Calibre 的电子书编辑功能修改输出文件的 CSS 文件。

**Reply by geekhuashan** [reply]
不知道如何操作啊，能有具体步骤吗。

翻译完了之后，output 就没有选项了啊，怎么修改啊

**Reply by bookfere** [reply, marked as answer]
这个需求同 #98，修改译文字体样式的功能将来可能会实现。

目前你需要用 Calibre 的电子书编辑器功能打开翻译好的电子书，然后打开任意一个 CSS 文件，然后在末尾添加以下样式，然后保存即可。

```css
*[lang=zh] {
    font-size: .8rem!important;
}
```

注意 `lang=zh` 要和实际的译文段落属性对应，随便打开一个 XHTML 检查一下即可。
